gocourse03/main.go

```diff
@@ -0,0 +1,123 @@
+package main
+
+import (
+	"fmt"
+	"time"
+)
+
+const (
+	Chip        = "Chip"
+	Dale        = "Dale"
+	Cheaser     = "Cheaser"
+	Hackwrench  = "Hackwrench"
+	NorthSector = "North"
+	SouthSector = "South"
+	WestSector  = "West"
+	EastSector  = "East"
+	Center      = "Center"
+)
+
+var rodents [4]Rodent
+var westRodents []Rodent
+var eastRodents []Rodent
+var northRodents []Rodent
+var southRodents []Rodent
+var movements []Movement
+
+func main() {
+	rodents = [4]Rodent{
+		{ID: 1, Sector: &westRodents, RodentType: Chip, FromTo: [2]string{WestSector, WestSector}},
+		{ID: 2, Sector: &eastRodents, RodentType: Dale, FromTo: [2]string{EastSector, EastSector}},
+		{ID: 3, Sector: &southRodents, RodentType: Cheaser, FromTo: [2]string{SouthSector, SouthSector}},
+		{ID: 4, Sector: &northRodents, RodentType: Hackwrench, FromTo: [2]string{NorthSector, NorthSector}},
+	}
+
+	westRodents = append(westRodents, rodents[0])
+	eastRodents = append(eastRodents, rodents[1])
+	southRodents = append(southRodents, rodents[2])
+	northRodents = append(northRodents, rodents[3])
+
+	PrintLocations()
+
+	moveThroughtCenter(1, time.Now().Add(10), WestSector, EastSector)
+	moveThroughtCenter(2, time.Now().Add(11), EastSector, NorthSector)
+	moveThroughtCenter(2, time.Now().Add(12), NorthSector, SouthSector)
+	moveThroughtCenter(3, time.Now().Add(13), SouthSector, WestSector)
+
+	PrintLocations()
+}
+
+type Rodent struct {
+	ID         int
+	Sector     *[]Rodent
+	RodentType string
+	FromTo     [2]string
+}
+
+type Movement struct {
+	MoveTime time.Time
+	FromTo   [2]string
+}
+
+func moveThroughtCenter(id int, moveTime time.Time, from string, to string) {
```

Comment:
Одруківка в назві функції:
```suggestion
func moveThroughCenter(id int, moveTime time.Time, from string, to string) {
```